skypy/galaxy/spectrum.py

```diff
@@ -205,37 +205,37 @@ def kcorrect_spectra(redshift, stellar_mass, coefficients):
 
 
 def mag_ab(spectrum, bandpass, redshift=None):
-    r'''Compute absolute AB magnitude from spectrum and bandpass.
+    r'''Compute absolute AB magnitudes from spectra and bandpasses.
 
-    This function takes an *emission* spectrum and an observation bandpass and
-    computes the AB magnitude for a source at 10pc (i.e. absolute magnitude).
-    The emission spectrum can optionally be redshifted. The definition of the
+    This function takes *emission* spectra and observation bandpasses and
+    computes the AB magnitudes for sources at 10pc (i.e. absolute magnitudes).
```

Comment:
This was actually not correctly put. It is completely up to the spectra whether they are absolute, apparent, or something else entirely. Perhaps you could fix that?

```suggestion
    computes the AB magnitudes.
```